modules/StreamingAPI/src/main/java/org/gephi/streaming/api/Graph2EventListener.java

```diff
@@ -0,0 +1,253 @@
+//                Map<String, Object> attributes = new HashMap<String, Object>();
+//                for (AttributeValue value: entry.getValue()) {
+//                    if (value.getColumn().getIndex() != PropertiesColumn.NODE_ID.getIndex()
+//                       && value.getColumn().getIndex() != PropertiesColumn.EDGE_ID.getIndex())
+//                        attributes.put(value.getColumn().getTitle(), value.getValue());
+//                }
+//                
+//                if (!attributes.isEmpty()) {
+//                    org.gephi.streaming.api.event.GraphEvent streamingEvent =
+//                            eventBuilder.graphEvent(ElementType.NODE, EventType.CHANGE, entry.getKey(), attributes);
+//                    eventHandler.handleGraphEvent(streamingEvent);
+//                }
+//            }
+//
+//            for (Map.Entry<String, List<AttributeValue>> entry: edgeChangeTable.entrySet()) {
+//                Map<String, Object> attributes = new HashMap<String, Object>();
+//                for (AttributeValue value: entry.getValue()) {
+//                    if (value.getColumn().getIndex() != PropertiesColumn.NODE_ID.getIndex()
+//                       && value.getColumn().getIndex() != PropertiesColumn.EDGE_ID.getIndex())
+//                        attributes.put(value.getColumn().getTitle(), value.getValue());
+//                }
+//
+//                if (!attributes.isEmpty()) {
+//                    org.gephi.streaming.api.event.GraphEvent streamingEvent =
+//                            eventBuilder.graphEvent(ElementType.EDGE, EventType.CHANGE, entry.getKey(), attributes);
+//                    eventHandler.handleGraphEvent(streamingEvent);
+//                }
+//            }
+//
+//            break;
+//        }
+    }
+
+    private Map<String, Object> getNodeAttributes(Node node) {
+        Map<String, Object> attributes = new HashMap<String, Object>();
+        for (Column column: node.getAttributeColumns()) {
+            if (column.getIndex() != GraphStoreConfiguration.ELEMENT_ID_INDEX) {
+                Object value = node.getAttribute(column);
+                if (value != null)
+                    attributes.put(column.getTitle(), value);
+            }
+        }
+
+        if (sendVizData) {
+            attributes.put("x", node.x());
+            attributes.put("y", node.y());
+            attributes.put("z", node.z());
+
+            attributes.put("r", node.r());
+            attributes.put("g", node.g());
+            attributes.put("b", node.b());
+
+            attributes.put("size", node.size());
+        }
+
+        return attributes;
+    }
+
+    private Map<String, Object> getEdgeAttributes(Edge edge) {
+        Map<String, Object> attributes = new HashMap<String, Object>();
+        for (Column column: edge.getAttributeColumns()) {
+            if (column.getIndex() != GraphStoreConfiguration.ELEMENT_ID_INDEX) {
```

Comment:
Same here, with `!column.isProperty()` you would ignore the id, the label, the weight and the timeset if defined